### DISCUSSION TITLE
Ignoring Hidden Files When Using `Storage::allFiles()`

### DISCUSSION THREAD

**Discussion by devajmeireles** [laravel/framework#48298] [Q&A] [answered]
It would be more than ideal for `Storage::allFiles()` to be capable of ignoring hidden files: `Storage::allFiles(hidden: false)`

_"Hidden file is a file, blah blah."_ 
- File Facade - **undocumented I guess**, has this option, **why** Storage can not have the same?

**Comment by shaedrich** [comment, marked as answer]
I suppose, that's not implemented in [League's](https://thephpleague.com) [Flysystem](https://flysystem.thephpleague.com/docs/usage/filesystem-api/). Maybe you can [file a feature request over there](https://github.com/thephpleague/flysystem/issues).
